src/shared/functions.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python3
+
+"""
+Takes a releases trigger file and created a mapping of all the
+OCI tags that are marked to be release.
+"""
+
+import json
+import os
+import re
+import subprocess
+from collections import defaultdict
+import yaml
+from src.image.utils.schema.triggers import ImageSchema, KNOWN_RISKS_ORDERED
+
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+class BadChannel(Exception):
+    """Error validating release channel."""
+
+
+def get_all_revision_tags(file_all_revision_tags):
+    """
+    This function permits to get all the revision tags.
+    """
+    with open(file_all_revision_tags, encoding="UTF-8") as rev_tags_f:
+        return rev_tags_f.read().strip().rstrip(",").lstrip(",").split(",")
+
+
+def get_all_releases(file_all_releases):
+    """
+    This function permits to get all the releases.
```

Comment:
all the docstrings should be fixed cause they are not useful. Example:

```python
def get_foo():
    """ Permits to get foo """
```

This isn't helpful and is just stating the obvious. the docstring should complement the function name and provide additional understanding that one can not get by just reading the fn name

Reply:
Does the news docstring is good?